packages/lib/services/noteList/defaultMultiColumnsRenderer.ts

```diff
@@ -108,7 +108,12 @@ const renderer: ListRenderer = {
 			`
 			{{#note.is_todo}}
 				<div class="checkbox">
-					<input data-id="todo-checkbox" type="checkbox" {{#note.todo_completed}}checked="checked"{{/note.todo_completed}}>
+					<input
+						data-id="todo-checkbox"
+						type="checkbox"
+						aria-label="{{note.todoStatusText}}"
+						tabindex="-1"
```

Comment:
Without `tabindex="-1"`, it's possible to focus just the checkbox in some cases. (For example, when navigating from the note title input with <kbd>shift</kbd>-<kbd>tab</kbd>).